File: Lexer/BreakToken.cpp
#include "token.h"
#include <string>
using namespace std;

class Break : public Token {
	bool Token::equals()
	{

	}

	int Token::hashCode()
	{
		return 11;
	}

	string Token::toString()
	{
		return "break";
	}
};

File: src/main/java/parser/BreakToken.java
package parser;

public class BreakToken implements Token {
	public boolean equals(final Object other) {
		return other instanceof BreakToken;
	}
	
	public int hashCode() {
		return 14;
	}
	
	public String toString() {
		return "BreakToken";
	}

}
